CHANGES.md

```diff
@@ -1,6 +1,38 @@
 
 # Changes
 
+## edge-21.1.2
+
+This edge release continues the work on de-coupling non-core Linkerd components.
+Commands that use the viz-extension i.e `dashboard`, `edges`, `routes`,
+`stat`, `tap` and `top` are moved to the `viz` sub-command. These commands are still
+available under root but are marked depreciated and will be removed in a
+later stable release.
+
+This release also features proxy's dependencies upgrade to the
+Tokio v1 ecosystem.
+
+* Moves sub-commands that use the viz-extension under `viz`
```

Comment:
As per the [style guide](https://github.com/BuoyantIO/buoyant-internal/blob/main/ops/linkerd2-release/style-guide.md),  bullet point verbs should be in past tense

```suggestion
* Moved sub-commands that use the viz-extension under `viz`
```